### PR TITLE
Add variants of equipped headsets sprite

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR 
Alt variants added too.

**Screenshots**

image image image image

image image image image

**Changelog**

:cl:
- add: More variants of equipped headsets sprite

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml [hunk 1]
```diff
@@ -9,9 +9,31 @@
...
     state: icon_alt
   - type: Clothing
+    equippedPrefix: alt
     slots:
     - ears
 
+- type: entity
+  parent: ClothingHeadsetAlt
+  id: ClothingHeadsetAltCentCom
+  name: CentCom overear-headset
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Ears/headsets_alt.yml:19]:
```suggestion
  name: CentCom over-ear headset
```

File: Resources/Prototypes/Entities/Clothing/Ears/headsets.yml [hunk 2]
```diff
@@ -118,6 +130,8 @@
     - Medical
   - type: Sprite
     sprite: Clothing/Ears/Headsets/medical.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/medical.rsi
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Ears/headsets.yml:134]: these shouldnt actually be necessary

**Review by mirrorcult — COMMENTED** [review]
sprites look good

File: Resources/Prototypes/Entities/Clothing/Ears/headsets.yml (hunk 2 above)
Comment (Morb0) [reply Resources/Prototypes/Entities/Clothing/Ears/headsets.yml:134]: But it does overwrite the `sprite` of the parent and use their own sprite.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**